internal/commands/extension_inspect_test.go

```diff
@@ -0,0 +1,663 @@
+							"https://path/to/test/extension",
+							"REMOTE ARCHIVE:",
+							simpleExtensionOutputSection)
+
+						assert.TrimmedEq(outBuf.String(), expectedOutput)
+					})
+				})
+			})
+		})
+
+		when("inspecting a extension on the registry", func() {
+			it.Before(func() {
+				simpleInfo.Location = buildpack.RegistryLocator
+			})
+
+			when("using the default registry", func() {
+				it.Before(func() {
+					mockClient.EXPECT().InspectExtension(client.InspectExtensionOptions{
+						ExtensionName: "urn:cnb:registry:test/extension",
+						Daemon:        true,
+						Registry:      "default-registry",
+					}).Return(simpleInfo, nil)
+				})
+				it("succeeds", func() {
+					command.SetArgs([]string{"urn:cnb:registry:test/extension"})
+					assert.Nil(command.Execute())
+
+					expectedOutput := fmt.Sprintf(inspectExtensionOutputTemplate,
+						"urn:cnb:registry:test/extension",
+						"REGISTRY IMAGE:",
+						simpleExtensionOutputSection)
+
+					assert.TrimmedEq(outBuf.String(), expectedOutput)
+				})
+			})
+
+			when("using a user provided registry", func() {
+				it.Before(func() {
+					mockClient.EXPECT().InspectExtension(client.InspectExtensionOptions{
+						ExtensionName: "urn:cnb:registry:test/extension",
+						Daemon:        true,
+						Registry:      "some-registry",
+					}).Return(simpleInfo, nil)
+				})
+
+				it("succeeds", func() {
+					command.SetArgs([]string{"urn:cnb:registry:test/extension", "-r", "some-registry"})
+					assert.Nil(command.Execute())
+
+					expectedOutput := fmt.Sprintf(inspectExtensionOutputTemplate,
+						"urn:cnb:registry:test/extension",
+						"REGISTRY IMAGE:",
+						simpleExtensionOutputSection)
+
+					assert.TrimmedEq(outBuf.String(), expectedOutput)
+				})
+			})
+		})
+
+		when("a depth flag is passed", func() {
+			it.Before(func() {
+				complexInfo.Location = buildpack.URILocator
+
+				mockClient.EXPECT().InspectExtension(client.InspectExtensionOptions{
+					ExtensionName: "/other/path/to/test/extension",
+					Daemon:        true,
+					Registry:      "default-registry",
+				}).Return(complexInfo, nil)
+			})
+
+			it("displays detection order to specified depth", func() {
+				command.SetArgs([]string{"/other/path/to/test/extension", "-d", "2"})
+				assert.Nil(command.Execute())
+
+				assert.AssertTrimmedContains(outBuf.String(), depthExtensionOutputSection)
+			})
+		})
+	})
+
+	when("verbose flag is passed", func() {
+		it.Before(func() {
+			simpleInfo.Location = buildpack.URILocator
+			mockClient.EXPECT().InspectExtension(client.InspectExtensionOptions{
+				ExtensionName: "/another/path/to/test/extension",
+				Daemon:        true,
+				Registry:      "default-registry",
+			}).Return(simpleInfo, nil)
+		})
+
+		it("displays all mixins", func() {
+			command.SetArgs([]string{"/another/path/to/test/extension", "-v"})
+			assert.Nil(command.Execute())
+
+			assert.AssertTrimmedContains(outBuf.String(), simpleExtensionMixinOutputSection)
+		})
+	})
```

Comment:
This can also be removed, since there are no stacks, there are no mixins - so we don't need `--verbose`